src/engine/Match_tainting_mode.ml

```diff
@@ -541,6 +541,9 @@ let taint_config_of_rule ~per_file_formula_cache xconf file ast_and_errors
   ( {
       Dataflow_tainting.filepath = !!file;
       rule_id = fst rule.R.id;
+      track_control =
+        snd spec.sources
+        |> List.exists (fun (src : R.taint_source) -> src.source_control);
```

Comment:
Above (existing code), we have code that looks like

```
spec.sources |> snd
      |> Common.map (fun (src : Rule.taint_source) -> (src.source_formula, src)
```

and here we have

```
snd spec.sources
      |> List.exists (fun (src : R.taint_source) -> src.source_control)
```

The `|>` usage seems inconsistent. Which is recommended?

Also, when do we fully spell out the module vs use its alias, i.e. `Rule.taint_source` vs `R.taint_source`?

Reply:
Yeah I'm not sure where it's best to put the `snd`, but I "fixed" it. I also replaced `Rule.` with `R.`, thanks for pointing it out! If there is a module alias then clearly we should be using it, or removing it.